app/charts/map/chart-map.tsx

```diff
@@ -55,7 +55,10 @@ export const ChartMapVisualization = ({
     variables: {
       locale,
       iri: dataSetIri,
-      measures: [areaDimensionIri, symbolDimensionIri],
+      measures: [
+        chartConfig.fields.areaLayer.measureIri,
+        chartConfig.fields.symbolLayer.measureIri,
+      ],
```

Comment:
What was the problem here ?

Reply:
Previously there were component iris (geo dimensions) and not measure dimensions in this array. Probably `measures` could be blank, as they are not used in this place, but in order to make advantage of `urql` caching, I modified this to match variables from `data-download` and `chart-options-selector` to reduce the number of requests.

Reply:
Hmm... After going through resolvers, it seems that `measures` prop is not even used :D

https://github.com/visualize-admin/visualization-tool/blob/main/app/graphql/resolvers.ts

Maybe it could be good to remove them altogether and clean up the code?

Reply:
Mmh it seems used here no https://github.com/visualize-admin/visualization-tool/blob/main/app/graphql/resolvers.ts#L184 ?

Reply:
Ah, I mean the `measures` prop in the observations query:

https://github.com/visualize-admin/visualization-tool/blob/main/app/graphql/queries/data-cubes.graphql

Reply:
It's not used in the resolver

Reply:
Sorry, I see that references to code lines weren't pasted in the links I sent...

Reply:
(updated the links) 😆

Reply:
Good point ! I think we can remove it.

Reply:
Not in this PR.